### PR TITLE
README: mention that Podman uses Buildah's API

### PR DESCRIPTION
**Pull request by vrothberg**
Signed-off-by: Valentin Rothberg <rothberg@redhat.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,7 +51,8 @@ build images.  The flexibility of building images without Dockerfiles allows
 for the integration of other scripting languages into the build process.
 Buildah follows a simple fork-exec model and does not run as a daemon
 but it is based on a comprehensive API in golang, which can be vendored
-into other tools.
+into other tools.  Podman uses Buildah's golang API for the podman-build
```
Comment (TomSweeneyRedHat) [inline README.md]: I think I'd put this new verbiage at the end of the following paragraph.  WDYT?

**Comment by TomSweeneyRedHat** [comment]
One question for consideration.  We'll need the final corresponding change in the Podman README.md too.  I've restarted the one flakey test.

File: README.md (hunk 1 above)
Comment (vrothberg) [reply README.md]: I put it here to have it close to the explanation of Buildah also being/exposing an golang API. But I can move it down :+1:.

Comment (rhatdan) [reply README.md]: It is also vendored into the OpenShift Image Builder.

Comment (rhatdan) [reply README.md]: How about 

For example,  Podman uses Buildah's golang API for the podman-build

Comment (TomSweeneyRedHat) [reply README.md]: just noted, missing a '.' at the end.  Rather than "podman-bulid", I think I'd go 
```
`podman build` command.
```

**Comment by vrothberg** [comment]
@rhatdan @TomSweeneyRedHat, I moved it one paragraph down and rephrased to be more explicit.

If you're cool, we can just force merge to avoid running the entire CI.

**Comment by rhatdan** [comment]
LGTM

**Pull request merged**